sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -79,6 +87,15 @@ static const struct snd_soc_acpi_link_adr tgl_rvp[] = {
 	{}
 };
 
+static const struct snd_soc_acpi_link_adr tgl_ref[] = {
```

Comment:
tgl_chromebook_base?

Reply:
Done.. Thanks for the review.

> tgl_chromebook_base?

Done.. Thanks for the review. re-posted.